deploy/sdk/src/dynamo/sdk/cli/utils.py

```diff
@@ -351,3 +351,20 @@ def resolve_service_config(
 
     logger.debug(f"Final resolved config: {service_configs}")
     return service_configs
+
+
+def set_deployment_target(target: str):
+    from dynamo.sdk.core.lib import set_target
+
+    if target == "kubernetes" or target == "bento":
```

Comment:
Is Bento meant to be an option? It's not listed/mentioned in the typer.Option description